WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController+Internal.h

```diff
@@ -20,7 +20,7 @@
 @property (nonatomic, strong) NSString                      *pushNotificationID;
 @property (nonatomic, strong) NSDate                        *pushNotificationDate;
 @property (nonatomic, strong) NSDate                        *lastReloadDate;
-@property (nonatomic, strong) NSMutableSet                  *notificationIdsMarkedForDeletion;
+@property (nonatomic, strong) NSMutableDictionary           *notificationDeletionBlocks;
```

Comment:
We actually need to **retain / copy** those blocks, since they come from another ViewController. Changing to MapTable should break things there (unless it's set to strong / strong, which really replicates NSDictionary's behaviour).

Thanks for checking!!